agents-api/test_api.py

```
@@ -3,82 +3,92 @@
-        "collection_name": "Test Collection from API"
+        "collection_name": "Test Collection from API",
     }
-    
+
     try:
         print("🧪 Testing Article Link Extractor...")
         print(f"📝 Request data: {json.dumps(test_data, indent=2)}")
-        
+
         # Make request
         response = requests.post(url, json=test_data)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)